tests/baselines/reference/tsxElementResolution10.errors.txt

```diff
@@ -18,9 +16,6 @@ tests/cases/conformance/jsx/file.tsx(19,2): error TS2322: Type '{ x: number; ren
     }
     var Obj1: Obj1type;
     <Obj1 x={10} />; // Error, no render member

```

Comment:
should this still be an error? The comment says so.

Reply:
The argument assignability error suppresses the construction error now. Specifically, since no signature matched the arguments provided, an error signature is returned with an `any`ish return type, which _is_ assignable to `ElementClass`.